### ISSUE TITLE
Transition class seems irrelevant when creating scale transitions

### ISSUE THREAD

**Issue by artursanntos** [tailwindlabs/tailwindcss.com#1636]
This issue looks both like a bug with tailwind and a documentation issue, here's why:

1. I was stylizing an image and was trying to scale it up when hovering. My class was: "transition duration-500 hover:scale-105".
2. The problem is that the duration property was not being applied because the duration property inside the transitions was overriding it.
3. I went to check the documentation and found this block in the documentation.
![Documentation](https://github.com/tailwindlabs/tailwindcss.com/assets/69633443/38ed206f-b73c-4434-ad0d-26d33d98cfd4)
4. So i noticed that my class was very similar to the code in the screenshot above. So i inspected the example buttons from the documentations and i noticed that these buttons **do not have the transition** property on their class.
5. So, my conclusion was that the transition property was unnecessary and exchanged it with only ease-in-out and it worked. 

The problem:
* Basically the documentation code snippet says that you should use "transition duration ease-in-out" and the example buttons to show how it works do not have the transition property in them.
* Also, i don't really know how to solve the problem with the transition overriding the duration property. This is not a major problem because using ease-in-out seems to do the job.

**Comment by thecrypticace** [comment]
You are correct that the transition property isn't strictly necessary — this is because the default value of `transition-property` is `all` which means that any property that's changed can be animated. The default duration is 0s — which is why things do not animate by default.

I'm not 100% certain why we omitted it from the actual preview — the behavior wouldn't change whether it was used or not in this case. So it may have just been "it was less typing". ¯\_(ツ)_/¯. The `transition` property is still a good one to use though because it's unlikely that you'd want _every_ property change animated once having set a duration but in some cases it doesn't matter if you're actually only changing just a few properties and expect them all to animate.

As for the duration from `transition` overriding a set duration — that should not happen — and _in my testing_ [it does not](https://play.tailwindcss.com/ecVZVWjMi7). The `duration` class is defined after the utility which ensures that the value from the duration utility takes precedence.

Do you have a repository I could look at that reproduces this situation?

**Comment by adamwathan** [comment]
Going to close this as it is recommended that you use the `transition` class as @thecrypticace mentioned because it limits transitions to properties that are generally considered safe to transition (stuff that isn't expensive and doesn't cause page layout to be recalculated).

For help with the issue of the duration being overridden I would recommend opening a discussion in our [discussion forum](https://github.com/tailwindlabs/tailwindcss/discussions), happy to help there!
